frontend/ui/elements/screensaver_menu.lua

```diff
@@ -159,6 +159,20 @@ return {
                         callback = function()
                             Screensaver:chooseFolder()
                         end,
+                        separator = true,
+                    },
+                    {
+                        text = _("Show images in alphabetical order"),
+                        help_text = _("When enabled, images will be shown in alphabetical order rather than in random order."),
```

Comment:
We display a single cover image, so it's not really "Show images in alpha order".
Pick image ?
May be you should mention the only benefit of this option: each image will be picked, no duplicate one before we iterate the whole set, something like that.